graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -12,31 +12,60 @@
 from ...tests import test_backend
 from ...tests.test_helpers import generate_schema, generate_schema_graph
 from ..test_helpers import SCHEMA_TEXT, compare_ignoring_whitespace, get_schema
-from .integration_backend_config import MATCH_BACKENDS, SQL_BACKENDS
+from .integration_backend_config import (
+    MATCH_BACKENDS, NEO4J_BACKENDS, REDISGRAPH_BACKENDS, SQL_BACKENDS
+)
 from .integration_test_helpers import (
-    compile_and_run_match_query, compile_and_run_sql_query, sort_db_results
+    compile_and_run_match_query, compile_and_run_neo4j_query, compile_and_run_redisgraph_query,
+    compile_and_run_sql_query, sort_db_results
 )
 
 
-# Store the test parametrization for running against all backends. Individual tests can customize
-# the list of backends to test against with the full @parametrized.expand([...]) decorator.
-all_backends = parameterized.expand([
+all_backends_list = [
     test_backend.ORIENTDB,
     test_backend.POSTGRES,
     test_backend.MARIADB,
     test_backend.MYSQL,
     test_backend.SQLITE,
     test_backend.MSSQL,
-])
+    test_backend.NEO4J,
+    test_backend.REDISGRAPH,
+]
+
 
 # Store the typical fixtures required for an integration tests.
 # Individual tests can supply the full @pytest.mark.usefixtures to override if necessary.
 integration_fixtures = pytest.mark.usefixtures(
+    'integration_neo4j_client',
     'integration_orientdb_client',
+    'integration_redisgraph_client',
     'sql_integration_data',
 )
 
 
+def use_all_backends(except_backends=()):
+    """Decorator for test functions to use specific backends.
+
+    By default, tests decorated with this function use all backends. However, some backends don't
+    support certain features, so it's useful to exclude certain backends for individual tests.
+
+    Args:
+        except_backends: Tuple[str], optional argument. Tuple of backend strings from
+                         test_backend.py to exclude in testing.
+
+    Returns:
+        function that expands tests. parameterized.expand() takes in a list of test parameters (in
```

Comment:
The mentioning of parameterized.expand() is best written as a comment instead of a docstring. A docstring should explain what the function does not how it does it.